doc/classes/@GDScript.xml

```diff
@@ -972,6 +972,19 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="smooth_step">
+			<return type="float">
+			</return>
+			<argument index="0" name="from" type="float">
+			</argument>
+			<argument index="1" name="to" type="float">
+			</argument>
+			<argument index="2" name="weight" type="float">
+			</argument>
+			<description>
+				Hermite interpolation between [code]from[/code] and [code]to[/code] using [code]weight[/code]. Simular to [code]lerp[/code] but interpolates faster at the beginning and slower at the end.
```

Comment:
```suggestion
				Hermite interpolation between [code]from[/code] and [code]to[/code] using [code]weight[/code]. Similar to [code]lerp[/code], but interpolates faster at the beginning and slower at the end.
```

Reply:
Done !

Reply:
@Calinou Please dont forget to approve changes ^^